bitnami/discourse/values.yaml

```diff
@@ -312,34 +316,46 @@ persistence:
   ##   GKE, AWS & OpenStack)
   ##
   # storageClass: "-"
-  ##
+
   ## If you want to reuse an existing claim, you can pass the name of the PVC using
   ## the existingClaim variable
   # existingClaim: your-claim
-  accessMode: ReadWriteOnce
+  ## To use the portal and to ensure you can scale Discourse you need to provide a
+  ## ReadWriteMany PVC, if you don't have a provisioner for this type of storage
+  ## we recommend that you install the NFS provisioner and map it to a RWO volume
+  ## helm install nfs-server stable/nfs-server-provisioner --set persistence.enabled=true,persistence.size=10Gi
+  ##
+  accessMode: ReadWriteMany
   size: 10Gi
 
 ## Kubernetes configuration
 ## For minikube, set this to NodePort, elsewhere use LoadBalancer or ClusterIP
 ##
 service:
   type: LoadBalancer
+
```

Comment:
Same as before, not a huge issue at all, but we usually tend to keep the components of a block together.

Reply:
done